src/Driver/Http2Driver.php

```diff
@@ -1633,7 +1660,8 @@ function (int $bodySize) use ($id) {
                 }
             }
         } catch (Http2ConnectionException $exception) {
-            yield $this->shutdown(null, $exception);
+            $this->shutdown(null, $exception);
```

Comment:
Why is this no longer awaited?

Reply:
Being in the parser, `yield` was causing `Client::close()` to not be called unless more data was received from the client (though a destructor probably called it anyway). Fixed in master as it isn't really relevant to this PR.